core/src/main/c/share/jit/avx2.h

```diff
@@ -105,28 +105,28 @@ namespace questdb::avx2 {
         return c.newConst(ConstPool::kScopeLocal, &broadcast_value, 32);
     }
 
+    // Reads length of variable size column with header stored in data vector (string, binary).
     jit_value_t read_mem_varlen(Compiler &c,
                                 uint32_t header_size,
                                 int32_t column_idx,
-                                const Gp &column_address,
-                                const Gp &varlen_indexes_ptr,
+                                const Gp &cols_ptr,
+                                const Gp &varsize_indexes_ptr,
```

Comment:
We should align the naming here. There seems to be some renaming that changed "varlen" to "varsize", but this function's name still uses "varlen". A few lines below, there's also the usage of the name "aux vector" to refer to the same thing that's otherwise called "varlen index".

Reply:
Yes, we use "var-size" everywhere in the code base, so that's why I decided to replace "var-len" with "var-size". Applied the suggestions in 2ac2b92f57fddfea41e1f9b03637fc460d4bce54